tests/model_fields/tests.py

```diff
@@ -661,6 +689,14 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+                qs = self.model.objects.filter(value__lte=min_value)
+                self.assertEqual(qs.count(), 1)
+                self.assertEqual(qs[0].value, min_value)
+            except (DataError, IntegrityError):
```

Comment:
Why is this exception catching needed? Could you add a comment about it? Try to minimize the try block and move everything else to an "else" block (try/except/else).